docs/repositories/repository-dashboard.md

```diff
@@ -65,12 +65,12 @@ Click **See all files** to open the [list of files](files.md) in the repository.
 
 ## Open pull requests
 
-The **Open pull requests** area displays the number of open pull requests in the repository and the split between the following status:
+The **Open pull requests** area displays the last updated pull requests and the split between the status of all open pull requests in your repository:
 
--   Not up to standards
--   Up to standards
--   Analyzing (intermediate status while Codacy is analyzing the pull request)
+-   **Up to standards:** Pull requests that meet the minimum quality levels
+-   **Not up to standards:** Pull requests that failed to meet at least one of the [quality gate rules defined for the repository](../repositories-configure/adjusting-quality-settings.md#gates)
+-   **Analyzing:** Pull requests currently being analyzed by Codacy
 
-Click **See all pull requests** to open the list of pull requests in the repository.
+Click a pull request to see the details of that pull request, or click **See all pull requests** to open the [list of pull requests](pull-requests.md) in the repository.
```

Comment:
We also have the interaction in the bars, to filter down the prs list, want to mention that?

Reply:
Good idea @inesgomas, I made an edit to take that into account. :+1: